blaze-client/src/main/scala/org/http4s/blaze/client/BlazeClient.scala

```diff
@@ -21,21 +21,19 @@ package client
 import cats.effect._
 import cats.effect.concurrent._
 import cats.effect.implicits._
-import cats.syntax.all._
-import java.net.SocketException
+import cats.implicits._
```

Comment:
Not a blocker, don't care.  May clean up if I do the merge.  I think it's getting late where @RafalSumislawski is.

Reply:
I must have added the `cats.implicits._` (Force of habbit) and then removed the `cats.syntax.all._` because it was unused. 

It is getting late, but I'm still up, just got sucked in some performance profiling.